MicroPython/requirements.txt

```diff
@@ -1,5 +1,5 @@
 wheel
 esptool==2.8
+gitpython
```

Comment:
missing version

Reply:
Added version to gitpython (skipped to wheel, as it is the same case as with setuptools - depending on minor version of Python)